MvvmCross/Binding/iOS/Target/MvxUITextViewTextTargetBinding.cs

```diff
@@ -46,7 +46,16 @@ public override void SubscribeToEvents()
                 return;
             }
 
-            target.Changed += this.EditTextOnChanged;
+			var textStorage = target.LayoutManager?.TextStorage;
+
+			if (textStorage == null)
+			{ 
+			    MvxBindingTrace.Trace(MvxTraceLevel.Error,
+						  "Error - NSTextStorage of UITextView is null in MvxUITextViewTextTargetBinding");
+				return;
+			}
+
+            textStorage.DidProcessEditing += this.EditTextOnChanged;
```

Comment:
There is a bit of confusion there I think, there used to be a method `textStorageDidProcessEditing` which is deprecated and replaced with a new class `NSTextStorageDelegate` which has a `DidProcessEditing` function. The later is the one I am using in the PR.